spec/pb_kits/playbook/kits/layout_spec.rb

```diff
@@ -22,7 +22,7 @@
                       .with_values("light", "dark", "gradient")
                       .with_default("light") }
   it { is_expected.to define_enum_prop(:layout)
-                      .with_values("sidebar", "collection", "kanban")
+                      .with_values("sidebar", "collection", "collection_detail", "kanban")
                       .with_default("sidebar") }
 
   describe "#classname" do
```

Comment:
@evalouderback Add a test to include collection_detail in the class name kinda like what has been done for collection already